src/main/java/org/candlepin/resource/validation/DTOValidator.java

```diff
@@ -14,21 +14,12 @@
  */
 package org.candlepin.resource.validation;
 
-import com.google.inject.Inject;
-
+import javax.validation.Validator;
```

Comment:
This will probably fail checkstyle too. The import order config needs javax imports after java imports: https://github.com/candlepin/candlepin/blob/358b5d0f790265a36591811b1c6f26fada64d122/config/checkstyle/checkstyle.xml#L202-L209